src/briefcase/console.py

```diff
@@ -208,16 +213,20 @@ def _log(
                 )
 
     def debug(self, message="", *, prefix="", markup=False):
-        """Log messages at debug level; included if verbosity>=2."""
+        """Log messages at debug level; included if verbosity >= 1."""
         self._log(
             preface=self.DEBUG_PREFACE,
             prefix=prefix,
             message=message,
-            show=self.verbosity >= self.DEBUG,
+            show=self.is_debug,
             markup=markup,
             style="dim",
         )
 
+    def info_verbose(self, message="", *, prefix="", markup=False):
+        """Log message at info level if debug enabled; included if verbosity >= 1."""
+        self._log(prefix=prefix, message=message, show=self.is_debug, markup=markup)
+
```

Comment:
I'm wondering if the issue here isn't that debug should be "deep_debug", and "info_verbose" should be "debug"...